dnd-bot/dnd_bot/tests/test_database_entity.py

```diff
@@ -33,7 +33,9 @@ def test_add_entity_no_id_game(postgresql):
     DatabaseConnection.add_to_db(f'INSERT INTO public."Game" (token) VALUES (1)', None, "element")
     id_entity = DatabaseEntity.add_entity("test_entity", x=1, y=2, description="test_description")
 
-    assert id_entity is None
+    assert id_entity == 1
+    # TODO should be None!! but somehow this change breaks all tests
```

Comment:
Jeśli dobrze rozumiem to tutaj jest jedynka dlatego że w bazie danych przy id_game w tabeli Entity nie ma constrainta NOT NULL, więc wszystko jest zgodnie z oczekiwaniami. Możliwe że ten costraint byłby tam przydatny, w takim razie należy wystawić odpowiednie issue.

Reply:
sprawdzanie, czy podajemy jakiekolwiek sensowne id_game w funkcji `add_entity` wywala dużą ilość testów, więc na razie zostawiłem takie coś na później. Najpewniej będzie trzeba dodać takie sprawdzanie do innych funkcji z modułu database